projects/public-app-getting-started-template/src/app/public-app.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "Get started with public apps",
+  "uid": "get-started-public-app",
+  "description": "An example to demonstrate of how to build a public app with developer projects.",
+  "allowedUrls": ["https://api.your-domain.com"],
```

Comment:
Should `allowedUrls` be empty to start? This would unintentionally give the CRM card access to the `api.your-domain.com` domain

Reply:
I'm going to ask this and the following Q about the webhook property in Slack to get a consensus.

Reply:
Resolved in https://github.com/HubSpot/hubspot-project-components/pull/21/commits/0bbb7b5be40f33949c4b2c5d71204e252d025bea